js_modules/dagster-ui/packages/ui-core/src/assets/useReportEventsModal.tsx

```diff
@@ -222,46 +238,14 @@ const ReportEventDialogBody = ({
             </Box>
           }
         >
-          {selections.map((range, idx) => (
-            <Box
-              key={range.dimension.name}
-              border="bottom"
-              padding={{vertical: 12, horizontal: 20}}
-            >
-              <Box as={Subheading} flex={{alignItems: 'center', gap: 8}}>
-                <Icon name="partition" />
-                {range.dimension.name}
-              </Box>
-              <Box>
-                Select partitions to materialize.{' '}
-                {range.dimension.type === PartitionDefinitionType.TIME_WINDOW
-                  ? 'Click and drag to select a range on the timeline.'
-                  : null}
-              </Box>
-
-              <DimensionRangeWizard
-                partitionKeys={range.dimension.partitionKeys}
-                health={{
-                  ranges: assetHealth.rangesForSingleDimension(
-                    idx,
-                    selections.length === 2 ? selections[1 - idx]!.selectedRanges : undefined,
-                  ),
-                }}
-                dimensionType={range.dimension.type}
-                selected={range.selectedKeys}
-                setSelected={(selectedKeys) =>
-                  setSelections((selections) =>
-                    selections.map((r) =>
-                      r.dimension === range.dimension ? {...r, selectedKeys} : r,
-                    ),
-                  )
-                }
-                partitionDefinitionName={range.dimension.name}
-                repoAddress={repoAddress}
-                refetch={async () => setLastRefresh(Date.now())}
-              />
-            </Box>
-          ))}
```

Comment:
This code was identical to the content of `DimensionRangeWizards`, I think I just forgot there was another copy of it in this modal